fix

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Not sure how long we will need this fix, maybe add a comment line "JDK-8215291" here? Although it's also possible to use git log...

Reply:
It fixes the 4.3.2 javadoc in-place and hopefully never again! There is a fix for the javadoc production with Java11 in the POM.